subprojects/packagefiles/libgpiod/man/meson.build

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+if get_option('manpages').enabled()
+    help2man_prog = find_program('help2man')
+    tools = [ gpiodetect, gpioinfo, gpioget, gpioset, gpiofind, gpiomon ]
+    mandir = join_paths(get_option('prefix'), get_option('mandir'))
+    mandirn = mandir + '1'
```

Comment:
This adds a "1" to "/usr/share/man", what you actually want is the install dir "/usr/share/man/man1"